Makefile

```
@@ -54,6 +54,9 @@ docker-run: docker 	## start the built docker image in a container
 		-e PATCHES_HOST=$(PATCHES_HOST) -e PRIVATE_KEY="id_rsa" \
 		--name $(APP_NAME) $(REGISTRY)/$(APP_NAME):$(TAG)
 
+docker-push: tmp docker
```

I don't think `tmp` is a target anymore.
```suggestion
docker-push: docker
```